### PR TITLE
Add README page for Microsoft Bookings configuration

### PR DESCRIPTION
**Pull request by ralphgabrinao**
# What
Add README page for Microsoft Bookings configuration

# Why
To add detailed explanation on how to configure Microsoft Bookings. The doc will also be referenced in the Sample itself in case no Bookings link is configured.

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) documentation.

**Does this PR contain ARM Template changes?**

- [ ] I have verified deploying using the ARM Template
- [ ] I have tested the deployed app end to end

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: docs/microsoft-bookings.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# How to Configure Microsoft Bookings
+
+This sample uses takes advantage of the Microsoft 365 Bookings app to power the consumer scheduling experience and create meetings for providers. Follow the steps below to create a Microsoft Bookings calendar and use it in the sample.
```
Comment (rvenugopal-msft) [inline docs/microsoft-bookings.md:3]: remove uses in the sentence

File: docs/microsoft-bookings.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Steps
+
+- [Create a Microsoft Bookings Calendar](#bookingscalendar)
```
[the remark this answers is not in the document]
Comment (rvenugopal-msft) [reply docs/microsoft-bookings.md:7]: @jjsanchezms how do you think?

File: docs/microsoft-bookings.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Click `Create calendar`.
+
+## <a id="bookingsurl">Set the Microsoft Bookings URL in the Sample</a>
+
+Copy the Microsoft Bookings URL from your Bookings calendar.
+
+![bookings-page-url.png](../docs/images/bookings/bookings-page-url.png)
+
+### Configure using environment variables
```
Comment (rvenugopal-msft) [inline docs/microsoft-bookings.md:34]: can we also add here on how to configure in app settings of deployed sample? Although its environment variables, it not very clear where this needs to be set.

Comment (rvenugopal-msft) [reply docs/microsoft-bookings.md:34]: Let's provide the link to make it more clear https://learn.microsoft.com/en-us/azure/app-service/configure-common?source=recommendations&tabs=portal

File: docs/microsoft-bookings.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+```
+VV_MICROSOFT_BOOKINGS_URL = "Your Bookings URL"
+```
+
+If your app is hosted in Azure App Service, you can set environment variables in the App Service's Application Settings. [Learn more about Azure App Service Application Settings.](https://learn.microsoft.com/en-us/azure/app-service/configure-common?tabs=portal)
```
Comment (rvenugopal-msft) [inline docs/microsoft-bookings.md:40]: remove the locale from the urls, so /en-us to be removed

**Review by rvenugopal-msft — APPROVED** [review]

**Review by hansung-msft — APPROVED** [review]

**Pull request merged**